pkg/process/net/common_linux.go

```diff
@@ -15,15 +15,16 @@ import (
 )
 
 const (
-	pingURL              = "http://unix/" + string(sysconfig.PingModule) + "/ping/"
-	tracerouteURL        = "http://unix/" + string(sysconfig.TracerouteModule) + "/traceroute/"
-	connectionsURL       = "http://unix/" + string(sysconfig.NetworkTracerModule) + "/connections"
-	procStatsURL         = "http://unix/" + string(sysconfig.ProcessModule) + "/stats"
-	registerURL          = "http://unix/" + string(sysconfig.NetworkTracerModule) + "/register"
-	statsURL             = "http://unix/debug/stats"
-	pprofURL             = "http://unix/debug/pprof"
-	languageDetectionURL = "http://unix/" + string(sysconfig.LanguageDetectionModule) + "/detect"
-	netType              = "unix"
+	pingURL               = "http://unix/" + string(sysconfig.PingModule) + "/ping/"
+	tracerouteURL         = "http://unix/" + string(sysconfig.TracerouteModule) + "/traceroute/"
+	connectionsURL        = "http://unix/" + string(sysconfig.NetworkTracerModule) + "/connections"
+	procStatsURL          = "http://unix/" + string(sysconfig.ProcessModule) + "/stats"
+	registerURL           = "http://unix/" + string(sysconfig.NetworkTracerModule) + "/register"
+	statsURL              = "http://unix/debug/stats"
+	pprofURL              = "http://unix/debug/pprof"
+	languageDetectionURL  = "http://unix/" + string(sysconfig.LanguageDetectionModule) + "/detect"
+	discoveryOpenPortsURL = "http://unix/" + string(sysconfig.DiscoveryModule) + "/open_ports"
```

Comment:
Let's add USM as codeowners for the file

Reply:
Is this endpoint already in system-probe?

Reply:
it is added in this PR